tests/strategies/filter/test_crop_filter.py

```diff
@@ -0,0 +1,42 @@
+"""Tests the crop filter strategy."""
+import os
+from pathlib import Path
+
+
+def test_crop_filter():
+    """Test the crop filter strategy on 'sample_1280_853.jpeg',
+    downloaded from filesamples.com (called 'sample_1280x853').
+
+    Note: This test incorporates much of the contents of the test
+    'test_jpeg.py', so if that test fails, this one should fail too.
+    """
+    from oteapi.models.filterconfig import FilterConfig
+    from oteapi.models.resourceconfig import ResourceConfig
+    from oteapi.strategies.filter.crop_filter import CropFilter
+    from oteapi.strategies.parse.image import ImageDataParseStrategy
+
+    parent_dir = Path(__file__).resolve().parents[1]
+    crop = [200, 300, 900, 700]
+    filter_config = FilterConfig(
+        filterType="filter/crop",
+        configuration={"crop": crop},
+    )
+    crop_filter_data = CropFilter(filter_config).get()
+    image_config = ResourceConfig(
+        downloadUrl="file://dummy",
+        mediaType="image/jpeg",
+        configuration={
+            "localpath": str(parent_dir),
+            "filename": "sample_1280_853.jpeg",
+            "crop": crop_filter_data["imagecrop"],
+        },
+    )
+    parser_jpeg = ImageDataParseStrategy(image_config)
+    parser_jpeg.get()
+
+    with open(parent_dir / "sample_700_400.jpeg", "rb") as sample:
+        target_data = sample.read()
+    with open(parent_dir / "cropped_sample_1280_853.jpeg", "rb") as cropped:
+        cropped_data = cropped.read()
+    os.remove(parent_dir / "cropped_sample_1280_853.jpeg")
```

Comment:
Put this in a `try/finally` block, where this one is in the `finally` part - first checking that the file exists, of course :) Note, you can also use the `unlink()` method of a `Path` object to achieve the same result - then there's no need to import `os`.

Reply:
For similar cases it might be worth using the `tmp_path` built-in pytest fixture as well ([docs](https://docs.pytest.org/en/7.0.x/reference/fixtures.html#reference-fixtures)).